cobalt/user_agent_platform_info.h

```diff
@@ -0,0 +1,140 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COBALT_USER_AGENT_PLATFORM_INFO_H_
+#define COBALT_USER_AGENT_PLATFORM_INFO_H_
+
+#include <map>
+#include <string>
+
+namespace cobalt {
+
+void GetUserAgentInputMap(
```

Comment:
nit: If this is only used in user_agent_platform_info.cc then it doesn't need to be declared here.

Reply:
`GetUserAgentInputMap` is also used in user_agent_platform_info_unittest.cc.

Reply:
Ah, gotcha thanks!

I searched in the code, but Github doesn't expand large files by default. My bad!